web/themes/custom/bevan/sass/components/_button.scss

```diff
@@ -14,7 +14,7 @@
 	border-color: $grey;
 	color: $grey;
 	background: $yellow;
-	outline: 0;
+	outline: 2px solid $grey;
```

Comment:
I thought this was supposed to be 3px?

Angela's comment in the ticket:
> The only thing I see here is that the outline in the designs is thicker (3px) compared to the build (1px).

Reply:
The border is 1px and the outline sits outside of the border, together they are 3px.

Reply:
Right, got ya. Makes sense. Thanks.